src/Runtime.Base/src/System/Runtime/InternalCalls.cs

```diff
@@ -397,12 +397,15 @@ internal static long RhGetGcTotalMemory()
         [DllImport(Redhawk.BaseName, CallingConvention = CallingConvention.Cdecl)]
         internal static extern void RhpSignalFinalizationComplete();
 
+        [PInvokeStaticResolution]
```

Comment:
We want all MRT imports to be statically resolved to work with static linking...